test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_torch_inductor_inference.py

```diff
@@ -0,0 +1,210 @@
+    return boto3.session.Session(region_name=region)
+
+
+def get_cloudwatch_client(region="us-east-1"):
+    """Get AWS CloudWatch client object. Currently assume region is IAD (us-east-1)"""
+    return get_boto3_session(region=region).client("cloudwatch")
+
+
+def put_metric_data(metric_name, namespace, unit, value, dimensions):
+    """Puts data points to cloudwatch metrics"""
+    cloudwatch_client = get_cloudwatch_client()
+    current_timestamp = datetime.datetime.utcnow()
+    try:
+        response = cloudwatch_client.put_metric_data(
+            Namespace=namespace,
+            MetricData=[
+                {
+                    "MetricName": metric_name,
+                    "Dimensions": dimensions,
+                    "Value": value,
+                    "Unit": unit,
+                    "Timestamp": current_timestamp,
+                }
+            ],
+        )
+    except ClientError as e:
+        LOGGER.error(
+            "Error: Cannot put data to cloudwatch metric: {}".format(metric_name))
+        LOGGER.error("Exception: {}".format(e))
+        raise e
+
+
+def read_metric(model_suite, csv_file):
+    csv = os.path.join("./", f"logs_{model_suite}", csv_file)
+    df = pd.read_csv(csv)
+    value = df[df.columns[-1]].iloc[0]
+    if isinstance(value, str):
+        for i in range(len(value)):
+            if value[i].isdecimal() or value[i] == ".":
+                continue
+            else:
+                return float(value[:i])
+    return float(value)
+
+
+def upload_metric(instance_type, precision, suite, metric_name, value, unit):
+    put_metric_data(
+        metric_name=metric_name,
+        namespace=f"PyTorch/EC2/Benchmarks/TorchDynamo/Inductor",
+        unit=unit,
+        value=value,
+        dimensions=unique_metric_dims(instance_type, precision, suite),
+    )
+
+
+@pytest.mark.parametrize("ec2_instance_type", ["c5.4xlarge", "m5.4xlarge"], indirect=True)
+@pytest.mark.parametrize("suite", ["huggingface", "timm", "torchbench"])
+@pytest.mark.parametrize("precision", ["float32"])
+def test_performance_ec2_pytorch_inference_cpu(suite, precision, pytorch_inference, ec2_connection, region):
+    _, image_framework_version = get_framework_and_version_from_tag(
+        pytorch_inference)
+    if Version(image_framework_version) in SpecifierSet("<2.0"):
+        ec2_performance_pytorch_inference(
+            pytorch_inference,
+            ec2_instance_type,
+            ec2_connection,
+            region,
+            suite,
+            precision,
+        )
+
+
+@pytest.mark.parametrize("ec2_instance_type", ["c6g.4xlarge", "c7g.4xlarge", "m7g.4xlarge"], indirect=True)
+@pytest.mark.parametrize("suite", ["huggingface", "timm", "torchbench"])
+@pytest.mark.parametrize("precision", ["float32"])
+@pytest.mark.parametrize("ec2_instance_ami", [UL20_CPU_ARM64_US_WEST_2], indirect=True)
+def test_performance_ec2_pytorch_inference_graviton(suite, precision, pytorch_inference, ec2_connection, region):
```

Comment:
Does DLC automatically skip this test when testing x86 container ? I don't see any markers that set this test apart from regular x86 tests.

It is likely that `pytorch_inference` always refers to the x86 container whereas `pytorch_inference_graviton` always refers to the graviton container.

@access2rohit  please confirm with @xncqr how this works.

Reply:
yes ... when graviton mode is enabled x86 tests are disabled. When its disabled only x86 tests are run (CPU/GPU)

Reply:
But you have not marked your tests to be run on specific processors. I can see from the test logs that currently both graviton and x86 tests are run in parallel.